app/src/main/java/com/github/onlynotesswent/ui/screen/AddNote.kt

```diff
@@ -129,9 +131,9 @@ fun AddNoteScreen(
 
               Button(
                   onClick = {
-                    if (saveButton == "Scan Image") {
+                    if (saveButton == "Take Picture") {
                       // call scan image API or functions. Once scanned, add the note to database
-
+                      scanner.scan()
```

Comment:
After scanner.scan() create a new note of Type.PDF and save it in database but don't need to save the pdf itself in the database